### PR TITLE
Merge stores after expression statements

### PR DESCRIPTION
**Pull request by d367wang**

### CODE REVIEW

File: dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java [hunk 1]
```diff
@@ -2591,7 +2679,9 @@ public Node visitErroneous(ErroneousTree tree, Void p) {
 
     @Override
     public Node visitExpressionStatement(ExpressionStatementTree tree, Void p) {
-        return scan(tree.getExpression(), p);
+        ExpressionTree exprTree = tree.getExpression();
```
Comment (d367wang) [inline dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java:2682]: undo the change

**Comment by wmdietl** [comment]
Replaced by #164
